Simple examples in Founders.rmd

RosCraddock opened the pull request:
Based on the same pedigree, provided simple examples considered in the 24/10 meeting as requested in #7. Additional text towards drafting the founders vignette, but still a work in progress.

RosCraddock committed:
Merge pull request #1 from AlphaGenes/devel

RosCraddock committed:
founderVignette

Draft of first part of founder.rmd with simple example covering multiple scenarios

RosCraddock committed:
Multiple traits and upgs

gregorgorjanc commented:
@RosCraddock will have a look. See what I have done in https://github.com/AlphaGenes/AlphaPart/commit/2f49d7fcd80245981c57057c99e296d63402480c regarding demonstrating how partitioning works - I know you are planning to do something similar for the metafounders.

RosCraddock committed:
Merge pull request #2 from AlphaGenes/devel

update

RosCraddock committed:
Update to founders.rmd

- More detail and theoretic justification.

RosCraddock committed:
Small clarification

RosCraddock committed:
Error handling & vignette update

Added error handling for unknown parent groups into AlphaPart and tested locally. This included:
-New parameter: UPGname for user to define the nomenclature used for UPGs (or metafounders). The default is "UPG".
- If statements covering potential user error.
- Updates to founder vignette
Will address the testing next.

RosCraddock committed:
Update documentation

RosCraddock commented:
Hi @gregorgorjanc,
Implemented some changes following Friday's meeting. To summarise:

**AlphaPart.R**
- New parameter: `UPGname` with default as "UPG". This gives users flexibility as to how the unknown parent groups/metafounders are named.
- Error handling of unknown parent groups/metafounders, considering whether each UPG have their own record, own path, both parents unknown, etc.

**Founder.rmd**
- Changes to inform users of the `UPGname`
- Clarification on what the UPG trends represent

**References.rmd**
- Two references linked to UPGs.

Next step:
Formal testing. Completed testing locally, but need to update and integrate into the testthat. To do so, will also review and update other tests, as some now issue a warning to consider centring or using UPGs.

RosCraddock committed:
Small corrections

Small corrections and striping out of `center=false` from testthat calls.

RosCraddock committed:
Merge pull request #5 from AlphaGenes/devel

Devel
